tests/unit/nupic/data/generators/anomalyzer_test.py

```diff
@@ -29,7 +29,7 @@
 import unittest2 as unittest
 
 from nupic.data.file_record_stream import FileRecordStream
-from nupic.research import anomalyzer
+from nupic.data.generators import anomalyzer
```

Comment:
This imports the module `anomalyzer`, not a class. There isn't actually an anomalyzer class. The module consists of some functions for doing transformations as well as code for when it gets executed directly as a script.